hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAMetadataOnly.java

```diff
@@ -460,13 +462,9 @@ public void testOMRetryCache() throws Exception {
 
   }
 
-  private void validateVolumesList(String userName,
-      Set<String> expectedVolumes) throws Exception {
-    ObjectStore objectStore = getObjectStore();
-
+  private void validateVolumesList(Set<String> expectedVolumes,
+      Iterator<? extends OzoneVolume> volumeIterator) throws Exception {
```

Comment:
Sorry, I missed it previously. I think this doesn't throws Exception any more. If so, can you drop the ``throws Exception`` part